snomed/com.b2international.snowowl.snomed.datastore.tests/src/com/b2international/snowowl/snomed/datastore/compare/ReferenceSetCompareTest.java

```diff
@@ -0,0 +1,224 @@
+/*******************************************************************************
+ * Copyright (c) 2020 B2i Healthcare. All rights reserved.
+ *******************************************************************************/
+package com.b2international.snowowl.snomed.datastore.compare;
+
+import static org.junit.Assert.assertEquals;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.junit.Test;
+
+import com.b2international.commons.exceptions.BadRequestException;
+import com.b2international.snowowl.core.compare.CompareSetResult;
+import com.b2international.snowowl.snomed.common.SnomedRf2Headers;
+import com.b2international.snowowl.snomed.common.SnomedTerminologyComponentConstants;
+import com.b2international.snowowl.snomed.core.domain.SnomedConcept;
+import com.b2international.snowowl.snomed.core.domain.refset.SnomedRefSetType;
+import com.b2international.snowowl.snomed.core.domain.refset.SnomedReferenceSet;
+import com.b2international.snowowl.snomed.core.domain.refset.SnomedReferenceSetMember;
+import com.b2international.snowowl.snomed.core.domain.refset.SnomedReferenceSetMembers;
+import com.google.common.collect.ImmutableList;
+
+/**
+ * @since 7.8
+ */
+public class ReferenceSetCompareTest {
```

Comment:
Rename this to `SnomedMapTypeReferenceSetCompareTest`.

Reply:
Moved generic compare tests to snowowl core tests.

Reply:
Done